site/go.mod

```diff
@@ -1,5 +1,5 @@
 module sigs.k8s.io/kueue/site
 
-go 1.21
+go 1.24
```

Comment:
This is because

```
3:35:18 PM: $ hugo --enableGitInfo --buildFuture -b $DEPLOY_PRIME_URL
3:35:18 PM: go: errors parsing go.mod:
3:35:18 PM: /opt/build/repo/site/go.mod:3: invalid go version '1.24.0': must match format 1.23
3:35:18 PM: Total in 9 ms
3:35:18 PM: Error: failed to load modules: failed to download modules: failed to execute 'go [mod download -modcacherw]': failed to execute binary "go" with args [mod download -modcacherw]: go: errors parsing go.mod:
3:35:18 PM: /opt/build/repo/site/go.mod:3: invalid go version '1.24.0': must match format 1.23
```

https://app.netlify.com/sites/kubernetes-sigs-kueue/deploys/67d27c7b2324d60008b92c82